code/game/machinery/computer/cloning.dm

```diff
@@ -209,8 +209,8 @@
 
 	data["feedback"] = feedback
 
-	if(feedback && feedback["color"] == "good")
```

Comment:
https://www.youtube.com/watch?v=wLg04uu2j2o

Reply:
In what cases are feedback not green but the scan succeeded?

Reply:
while the scanner was scanning and attempting to scan while already scanning

Reply:
Or whenever the patient has already been scanned, but I changed that in #25079 to be a green feedback